src/ol/source/common.js

```diff
@@ -12,7 +13,10 @@ export const DEFAULT_WMS_VERSION = '1.3.0';
  * Context options to disable image smoothing.
  * @type {Object}
  */
-export const IMAGE_SMOOTHING_DISABLED = {
-  imageSmoothingEnabled: false,
-  msImageSmoothingEnabled: false,
-};
+export const IMAGE_SMOOTHING_DISABLED =
+  FIREFOX || SAFARI || WEBKIT
```

Comment:
Not a good idea to assume IE when it's not FIREFOX or SAFARI or WEBKIT. Better to use feature detection, e.g.
```js
'imageSmoothingEnabled' in CanvasRenderingContext2D.prototype
```
But that should not be exported here. Better to create that object in the TileImage source constructor when needed, to avoid problems when OpenLayers is loaded in a non-browser environment.

Reply:
I am going to revert that change as the fix doesn't work.

Reply:
I have now realised `msImageSmoothingEnabled`  isn't relevant to the problem. While testing locally without it I had been applying `imageSmoothingEnabled: false` to the reprojection of all layers which did fix the problem, but only if you want all layers on both both maps to have smoothing disabled, so it looks like a bug in the current version of Chrome.  The original purpose of the PR to fix the reprojection edges which also applies other browsers is complete and the PR is ready for merging.